### ISSUE TITLE
欢迎分享CVPR 2023 论文和代码 / Welcome to share the paper and code of CVPR 2023

### ISSUE THREAD

**Issue by amusi** [amusi/CVPR2024-Papers-with-Code#166]
[The format of the issue]
Paper name/title:
Paper link:
Code link:

**Comment by VainF** [comment]
Paper title: DepGraph: Towards Any Structural Pruning
Paper link: https://arxiv.org/abs/2301.12900
Code link: https://github.com/VainF/Torch-Pruning

Thank you!

**Comment by joellliu** [comment]
Paper title: PolyFormer: Referring Image Segmentation as Sequential Polygon Generation"
Paper link: https://arxiv.org/abs/2302.07387

Thank you!

**Comment by FingerRec** [comment]
Paper title: All in One: Exploring Unified Video-Language Pre-training
Paper link: https://arxiv.org/abs/2203.07303
Code link: https://github.com/showlab/all-in-one


Paper title: Position-guided Text Prompt for Vision Language Pre-training
Paper link: https://arxiv.org/abs/2212.09737
Code link: https://github.com/sail-sg/ptp

**Comment by tobran** [comment]
Paper title: GALIP: Generative Adversarial CLIPs for Text-to-Image Synthesis
Paper link: [https://arxiv.org/abs/2301.12959](https://arxiv.org/abs/2301.12959)
Code link: [https://github.com/tobran/GALIP](https://github.com/tobran/GALIP)

GALIP是一个简单，快速，高质量的文本到图像生成模型，对比需要数百张GPU，400M图文对，数周时间进行预训练的Diffusion Model和Autoregressive Model，GALIP仅需8张3090，12M图文对，3天的预训练时间，取得了相当甚至更好的效果，同时生成速度提高了120倍，支持仅CPU生成。代码和预训练模型已经开源。

GALIP is a simple, fast, and high-quality **Text-to-Image Generative Model** with comparable results to large pretrained Autoregressive and Diffusion models and 120 times faster synthesis speed. 
Compared with the Diffusion and Autoregressive models which require hundreds of GPUs, 400M image-text pairs, and several weeks for pre-training, GALIP only needs 8x3090, 12M image-text pairs, and 3 days for pre-training. Furthermore, GALIP can be used without GPU. The code and pre-trained models have been released.

**Comment by aminshabani** [comment]
Paper title: HouseDiffusion: Vector Floorplan Generation via a Diffusion Model with Discrete and Continuous Denoising
Paper link: https://arxiv.org/abs/2211.13287
Code link: https://github.com/aminshabani/house_diffusion

**Comment by GenjiB** [comment]
Vision Transformers are Parameter-Efficient Audio-Visual Learners
Project page: https://yanbo.ml/project_page/LAVISH/
code: https://github.com/GenjiB/LAVISH

**Comment by wuxiaolang** [comment]
3D Visual and Language
Paper title: EDA: Explicit Text-Decoupling and Dense Alignment for 3D Visual Grounding
Paper link: https://arxiv.org/abs/2209.14941
Code link: https://github.com/yanmin-wu/EDA

**Comment by Gaaaavin** [comment]
> Paper title: DeepMapping2: Self-Supervised Large-Scale LiDAR Map Optimization Paper link: https://arxiv.org/abs/2212.06331 Code link: https://github.com/ai4ce/DeepMapping2

This paper can be categorized into "3D point cloud"

**Comment by jiamingzhang94** [comment]
Paper title: Unlearnable Clusters: Towards Label-agnostic Unlearnable Examples
Paper link: https://arxiv.org/abs/2301.01217
Code link: https://github.com/jiamingzhang94/Unlearnable-Clusters

**Comment by realPasu** [comment]
Paper title: MM-Diffusion: Learning Multi-Modal Diffusion Models for Joint Audio and Video Generation
Paper link: https://arxiv.org/abs/2212.09478
Code link: https://github.com/researchmm/MM-Diffusion

**Comment by noahzn** [comment]
Paper title: Lite-Mono: A Lightweight CNN and Transformer Architecture for Self-Supervised Monocular Depth Estimation
Paper link: https://arxiv.org/abs/2211.13202
Code link: https://github.com/noahzn/Lite-Mono

**Comment by MrGiovanni** [comment]
医学影像分析

**1. Label-Free Liver Tumor Segmentation**
Paper: https://arxiv.org/pdf/2210.14845.pdf
Code: https://github.com/MrGiovanni/SyntheticTumors

**2. Deep Feature In-painting for Unsupervised Anomaly Detection in X-ray Images**
Paper: https://arxiv.org/pdf/2111.13495.pdf
Code: https://github.com/tiangexiang/SQUID

Thank you!

**Comment by ymy-k** [comment]
Paper title: DeepSolo: Let Transformer Decoder with Explicit Points Solo for Text Spotting
Paper link: https://arxiv.org/pdf/2211.10772v3.pdf
Code link: https://github.com/ViTAE-Transformer/DeepSolo

Thank you!

**Comment by VainF** [comment]
Paper title: DepGraph: Towards Any Structural Pruning
Paper link: https://arxiv.org/abs/2301.12900
Code link: https://github.com/VainF/Torch-Pruning

Thank you! This paper should be categorized as ``Network Pruning``

**Comment by Jialing-Zhang** [comment]
Paper title: Back to the Source: Diffusion-Driven Adaptation to Test-Time Corruption
Paper link: https://arxiv.org/abs/2207.03442
Code link: https://github.com/shiyegao/DDA

**Comment by limacv** [comment]
Paper title: 3D Video Loops from Asynchronous Input
Paper link: https://arxiv.org/abs/2303.05312
Project page: https://limacv.github.io/VideoLoop3D_web/
Code link: https://github.com/limacv/VideoLoop3D

This paper should be in a new category named ```新视点合成(Novel View Synthesis)```, which I believe is also a hot topic with many more papers. But it can also be categorized as NeRF if no more sections can be added. Thank you!

**Comment by rayleizhu** [comment]
Paper name/title: BiFormer: Vision Transformer with Bi-Level Routing Attention
Paper link: Coming soon
Code link: https://github.com/rayleizhu/BiFormer

**Comment by 2y7c3** [comment]
Paper title: Super-Resolution Neural Operator
Paper link: https://arxiv.org/abs/2303.02584
Code link: https://github.com/2y7c3/Super-Resolution-Neural-Operator

**Comment by stdKonjac** [comment]
Paper name/title: Learning Transferable Spatiotemporal Representations from Natural Script Knowledge
Paper link: https://arxiv.org/abs/2209.15280
Code link: https://github.com/TencentARC/TVTS

**Comment by Carlyx** [comment]
Paper name/title: DPE: Disentanglement of Pose and Expression for General Video Portrait Editing
Paper link: https://arxiv.org/abs/2301.06281
Code link: https://carlyx.github.io/DPE/

**Comment by vinthony** [comment]
paper name/title:  SadTalker： Learning Realistic 3D Motion Coefficients for Stylized Audio-Driven Single Image Talking Face Animation
Paper link: https://arxiv.org/abs/2211.12194
Code link: https://github.com/Winfredy/SadTalker

**Comment by slacklife** [comment]
Paper name/title: DeepMAD: Mathematical Architecture Design for Deep Convolutional Neural Network
Paper link: https://arxiv.org/abs/2303.02165
Code link: https://github.com/alibaba/lightweight-neural-architecture-search

Please put it in the `backbone` chapter of the README.md.

**Comment by Haiyang-W** [comment]
Paper name/title: DSVT: Dynamic Sparse Voxel Transformer with Rotated Sets

**Comment by Yueming6568** [comment]
Paper title: DeltaEdit: Exploring Text-free Training for Text-driven Image Manipulation
Paper link: https://arxiv.org/abs/2303.06285
Code link: https://github.com/Yueming6568/DeltaEdit

Thank you : )
please put it in the GAN/CLIP/image manipulation/ image generation chapters.

**Comment by rayleizhu** [comment]
The Arxiv link for BiFormer is now available. Please update. Thanks!

Paper name/title: BiFormer: Vision Transformer with Bi-Level Routing Attention
Paper link: https://arxiv.org/abs/2303.08810
Code link: https://github.com/rayleizhu/BiFormer

**Comment by dingfengshi** [comment]
Paper title: TriDet: Temporal Action Detection with Relative Boundary Modeling
Paper link: https://arxiv.org/pdf/2303.07347.pdf
Code link: https://github.com/dingfengshi/TriDet

maybe it can be put it in Video Understanding or a new chapter Action Detection? Thank you!

**Comment by lixinustc** [comment]
Thanks for it.
Paper title: Causal-IR: Learning Distortion Invariant Representation for Image Restoration from A Causality Perspective
Paper link: https://arxiv.org/pdf/2303.06859.pdf
Code link: https://github.com/lixinustc/Casual-IR-DIL

The code will be released soon.

**Comment by l1997i** [comment]
Paper title: Less is More: Reducing Task and Model Complexity for 3D Point Cloud Semantic Segmentation
Paper link: [https://arxiv.org/abs/2303.11203](https://arxiv.org/abs/2303.11203)
Code link: [https://github.com/l1997i/lim3d](https://github.com/l1997i/lim3d)

The code will be released soon. Thanks in advance!

**Comment by ShirleyMaxx** [comment]
Paper name/title: GFPose: Learning 3D Human Pose Prior with Gradient Fields
Paper link: https://arxiv.org/pdf/2212.08641.pdf
Code link: https://github.com/Embracing/GFPose

Thank you!

**Comment by shikiw** [comment]
Paper name/title: Diversity-Aware Meta Visual Prompting
Paper link: https://arxiv.org/abs/2303.08138
Code link: https://github.com/shikiw/DAM-VP

Thanks a lot!

**Comment by XinyuSun** [comment]
Paper name/title: Masked Motion Encoding for Self-Supervised Video Representation Learning
Paper link: https://arxiv.org/abs/2210.06096
Code link: https://github.com/XinyuSun/MME

Thanks a lot!

**Comment by nihaomiao** [comment]
Paper name/title: Conditional Image-to-Video Generation with Latent Flow Diffusion Models
Paper link: https://arxiv.org/pdf/2303.13744.pdf
Code link: https://github.com/nihaomiao/CVPR23_LFDM

Thanks a lot!

**Comment by man805** [comment]
Paper title: Diffusion Video Autoencoders: Toward Temporally Consistent Face Video Editing via Disentangled Video Encoding
Paper link: https://arxiv.org/abs/2212.02802
Project page: https://diff-video-ae.github.io
Code link: https://github.com/man805/Diffusion-Video-Autoencoders

Thank you!

**Comment by sjtuxcx** [comment]
Paper title: EqMotion: Equivariant Multi-agent Motion Prediction with Invariant Interaction Reasoning
Paper link: https://arxiv.org/abs/2303.10876
Code link: https://github.com/MediaBrain-SJTU/EqMotion

Thanks a lot!

**Comment by dailenson** [comment]
Paper title: Disentangling Writer and Character Styles for Handwriting Generation
Paper link: https://arxiv.org/abs/2303.14736
Code link: https://github.com/dailenson/SDT
Thanks a lot!

**Comment by MrGiovanni** [comment]
医学影像分析

1. Label-Free Liver Tumor Segmentation
Paper: https://arxiv.org/pdf/2303.14869.pdf
Code: https://github.com/MrGiovanni/SyntheticTumors

2. Deep Feature In-painting for Unsupervised Anomaly Detection in X-ray Images
Paper: https://arxiv.org/pdf/2111.13495.pdf
Code: https://github.com/tiangexiang/SQUID

Thank you!

**Comment by lizhou-cs** [comment]
Paper title:Joint Visual Grounding and Tracking with Natural Language Specification
Paper link: https://arxiv.org/abs/2303.12027
Code link: https://github.com/lizhou-cs/JointNLT

**Comment by JunweiZheng93** [comment]
CVPR 2023 Highlight!
Title: Attention-based Point Cloud Edge Sampling
Link: https://arxiv.org/abs/2302.14673
Code: https://github.com/JunweiZheng93/APES
Project page: https://junweizheng93.github.io/publications/APES/APES.html

Thx!

**Comment by bmlklwx** [comment]
[CVPR2023 Highlight]
Paper name/title: Marching-Primitives: Shape Abstraction from Signed Distance Function
Paper link: https://arxiv.org/abs/2303.13190
Code link: https://github.com/ChirikjianLab/Marching-Primitives

**Comment by lkeab** [comment]
Paper name/title: Mask-Free Video Instance Segmentation
Paper link: https://arxiv.org/abs/2303.15904
Code link: https://github.com/SysCV/MaskFreeVis

**Comment by SiyuanYan1** [comment]
Paper name/title:Towards Trustable Skin Cancer Diagnosis via Rewriting Model's Decision
Paper link:https://arxiv.org/pdf/2303.00885.pdf

**Comment by youngLBW** [comment]
Paper name/title: A Hierarchical Representation Network for Accurate and Detailed Face Reconstruction from In-The-Wild Images
Paper link: https://arxiv.org/abs/2302.14434
Code link: https://younglbw.github.io/HRN-homepage/

**Comment by cwhgn** [comment]
Paper name/title: Beyond Appearance: a Semantic Controllable Self-Supervised Learning Framework for Human-Centric Visual Tasks
Paper link: https://arxiv.org/abs/2303.17602
Code link: https://github.com/tinyvision/SOLIDER

**Comment by HuiZhang0812** [comment]
Paper title: Prototypical Residual Networks for Anomaly Detection and Localization
Paper link: https://arxiv.org/abs/2212.02031

**Comment by kwonjunn01** [comment]
Paper name/title: Probabilistic Prompt Learning for Dense Prediction
Paper link: http://arxiv.org/abs/2304.00779

**Comment by azhuantou** [comment]
Paper title: Hierarchical Supervision and Shuffle Data Augmentation for 3D Semi-Supervised Object Detection
Paper link: https://arxiv.org/abs/2304.01464
Code link: https://github.com/azhuantou/HSSDA

Thank you : )
please put it in the point cloud/semi-supervised 3D object detection chapters.

**Comment by DerrickXuNu** [comment]
Paper tile :  V2V4Real: A Real-world Large-scale Dataset for Vehicle-to-Vehicle Cooperative Perception (Highlight paper)
Paper Link: https://arxiv.org/abs/2303.07601
Website link: https://mobility-lab.seas.ucla.edu/v2v4real/
Code Link:  https://github.com/ucla-mobility/v2v4real

Thanks you! Please put it under the dataset chapter.

**Comment by wenzhengzeng** [comment]
Paper name/title: Real-time Multi-person Eyeblink Detection in the Wild for Untrimmed Video
Paper link: https://arxiv.org/abs/2303.16053
Code link: https://github.com/wenzhengzeng/MPEblink

Thanks you! Please put it under the dataset or new task chapter.

**Comment by Little-Podi** [comment]
Paper title: Generalized Relation Modeling for Transformer Tracking
Paper link: https://arxiv.org/abs/2303.16580
Code link: https://github.com/Little-Podi/GRM
Thx in advance!

**Comment by yorkeyao** [comment]
Paper name/title: Large-scale Training Data Search for Object Re-identification
Paper link: https://arxiv.org/abs/2303.16186
Code link: https://github.com/yorkeyao/SnP

**Comment by JiaxiongQ** [comment]
Paper title: Looking Through the Glass: Neural Surface Reconstruction Against High Specular Reflections
Paper link: https://arxiv.org/abs/2304.08706
Code link: https://github.com/JiaxiongQ/NeuS-HSR

**Comment by Jun-CEN** [comment]
Paper title: Enlarge Instance-specific and Class-specific Information for Open-set Action Recognition
Paper link: https://arxiv.org/abs/2303.15467
Code link: https://github.com/Jun-CEN/PSL

**Comment by Woolseyyy** [comment]
NeRF
Paper title: NeFII: Inverse Rendering for Reflectance Decomposition with Near-Field Indirect Illumination
Paper link: https://arxiv.org/abs/2303.16617

**Comment by nandometzger** [comment]
Paper name: Guided Depth Super-Resolution by Deep Anisotropic Diffusion
Paper link: https://arxiv.org/abs/2211.11592
Code link: https://github.com/prs-eth/Diffusion-Super-Resolution

**Comment by LuPaoPao** [comment]
Paper name/title:  Neuron Structure Modeling for Generalizable Remote Physiological Measurement 
Paper link: https://arxiv.org/abs/2303.05955
Code link: https://github.com/LuPaoPao/NEST-rPPG

**Comment by ChenFengYe** [comment]
Category: 3D动作生成 (3D Motion Generation)
Executing your Commands via Motion Diffusion in Latent Space
Paper: https://arxiv.org/abs/2212.04048
Code: https://github.com/ChenFengYe/motion-latent-diffusion
Homepage: https://chenxin.tech/mld/

**Comment by XuZhengzhuo** [comment]
Paper name/title: Learning Imbalanced Data with Vision Transformers
Paper link: https://arxiv.org/abs/2212.02015
Code link: https://github.com/XuZhengzhuo/LiVT
Category: Long-Tail

**Comment by ZYK100** [comment]
Diverse Embedding Expansion Network and Low-Light Cross-Modality Benchmark for Visible-Infrared Person Re-identification

Paper: https://arxiv.org/abs/2303.14481
Code: https://github.com/ZYK100/LLCM
Dataset: https://github.com/ZYK100/LLCM/blob/main/LLCM%20Dataset%20Agreement/LLCM%20DATASET%20RELEASE%20AGREEMENT.pdf

**Comment by qiulingxu** [comment]
MEDIC: Remove Model Backdoors via Importance Driven Cloning 
Paper: https://openaccess.thecvf.com/content/CVPR2023/html/Xu_MEDIC_Remove_Model_Backdoors_via_Importance_Driven_Cloning_CVPR_2023_paper.html
Code: https://github.com/qiulingxu/MEDIC

**Comment by sunmenmian** [comment]
S3C: Semi-Supervised VQA Natural Language Explanation via Self-Critical Learning
Paper: 
[https://openaccess.thecvf.com/content/CVPR2023/html/Suo_S3C_Semi-Supervised_VQA_Natural_Language_Explanation_via_Self-Critical_Learning_CVPR_2023_paper.html](url)
Code:[https://github.com/Suo-Wei/S3C](url)

**Comment by yichen928** [comment]
Active Finetuning: Exploiting Annotation Budget in the Pretraining-Finetuning Paradigm
Paper: https://arxiv.org/abs/2303.14382
Code: https://github.com/yichen928/ActiveFT
